smallrye-reactive-messaging-gcp-pubsub/src/main/java/io/smallrye/reactive/messaging/gcp/pubsub/PubSubConnector.java

```diff
@@ -179,6 +179,10 @@ private static Path getCredentialPath(final Config config) {
     private static PubsubMessage buildMessage(final Message<?> message) {
         if (message instanceof PubSubMessage) {
             return ((PubSubMessage) message).getMessage();
+        } else if (message.getPayload() instanceof PubSubMessage) {
+            return ((PubSubMessage) message.getPayload()).getMessage();
+        } else if (message.getPayload() instanceof PubsubMessage) {
```

Comment:
I have a question just for my understanding. Here we now have two equal checks `else if (message.getPayload() instanceof PubSubMessage)`. So ` return ((PubsubMessage) message.getPayload());` would never be executed. Or am I wrong?

Reply:
Ah yes!

Reply:
Wait am I the one missing something ? Second if checks payload against `io.smallrye.reactive.messaging.gcp.pubsub.PubSubMessage` and the third one against `com.google.pubsub.v1.PubsubMessage`.

Reply:
Oh, there is a lowercase `s` in `PubsubMessage` in the third check. Sorry for the inconvenience :/